### ISSUE TITLE
Could you help me Why mAP and speed difference too much with paper?

### ISSUE THREAD

**Issue by bluesky1308** [david8862/keras-YOLOv3-model-set#111]
![yolo3-coco](https://user-images.githubusercontent.com/16258812/92899882-2b5d7300-f45a-11ea-9dd9-e90fe010a02e.jpg)
Dear Bro,

I tested yolo3 with yolo3.weight-darknet-53 on COCO2014; (hardware gtx1060-6gb).
However, My result AP higher than the paper 65.6 vs 55.3 mAP50; total AP higher too 36.6 vs 33.xx
and the other hand, my speed too slow: large scale image 5fps But my VGG-SSD300 is 35fps.

Thank you

**Comment by david8862** [comment]
@bluesky1308  the 55.3 mAP/33.xx AP for standard YOLOv3 is evaluate on COCO test set. So it may be different if you check with val data.

Regarding inference speed, since by default I just use numpy based postprocess and also the keras framework may not be so efficient as some others, so it may not performance well enough.

**Comment by bluesky1308** [comment]
@david8862 thank you so much.

Do you have anyway to improve speed. The performance issue impact only yolov3-darknet or both yolov3-mobilnet, yolov3-tiny and yolov4-xxx. 

Have a nice day!

**Comment by david8862** [comment]
@bluesky1308  maybe you can try to change [YOLO_np](https://github.com/david8862/keras-YOLOv3-model-set/blob/master/yolo.py#L376) to "YOLO" to apply the tf.keras postprocess model if you're using yolo.py to do live inference.

**Comment by bluesky1308** [comment]
@david8862  Thank you. I will try it.
